docs/DocGenerator/CodeParser.py

```diff
@@ -5,10 +5,14 @@
 import yaml
 from Config import Config
 from TestCaseParser import TestCaseParser
+from Utils import remove_inexistent
 from docstring_parser import parse
 from comment_parser import comment_parser
 import warnings
 
+INTERNAL_FIELDS = ['Id', 'Group Id', 'Name']
```

Comment:
Since we agreed to use snake case for those fields, we should change this.

Reply:
This changes were found and requested during the exploratory testing.
So they are included in this [PR](https://github.com/wazuh/wazuh-qa/pull/1680)